src/pubkey.h

```diff
@@ -206,6 +207,24 @@ class CPubKey
     bool Derive(CPubKey& pubkeyChild, ChainCode &ccChild, unsigned int nChild, const ChainCode& cc) const;
 };
 
+class XOnlyPubKey {
+private:
+    uint256 m_keydata;
+
+public:
+    /** Construct an x-only pubkey from exactly 32 bytes. */
+    XOnlyPubKey(Span<const unsigned char> input);
+
+    /** Verify a 64-byte Schnorr signature.
+     *
+     * If the signature is not exactly 64 bytes, false is returned.
```

Comment:
`VerifySchnorr` has `assert(sigbytes.size() == 64);` so false won't be returned. Should either change this comment or change the assertion to `if (sigbytes.size() != 64) return false;`

Reply:
Fixed.